### ISSUE TITLE
meson build fails with non-matching version number

### ISSUE THREAD

**Issue by umlaeute** [thestk/rtaudio#404]
with RtAudio-6.0.0 I do:

```sh
$ meson build
The Meson build system
Version: 1.2.0
Source dir: <<path>>/rtaudio
Build dir: <<path>>/rtaudio/build
Build type: native build
Project name: RtAudio
Project version: 6.0.0beta1
C++ compiler for the host machine: ccache c++ (gcc 13.1.0 "c++ (Debian 13.1.0-9) 13.1.0")
C++ linker for the host machine: c++ ld.bfd 2.40.90.20230720
Host machine cpu family: x86_64
Host machine cpu: x86_64

meson.build:30:0: ERROR: Assert failed: Meson's RtAudio version does not match the version in header file.

A full log can be found at <<path>>/rtaudio/build/meson-logs/meson-log.txt
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.
```

most likely because `RtAudio.h` says that the version is `6.0.0` https://github.com/thestk/rtaudio/blob/0e11b50a7ba85875d09d60c097092d1d4bd760b2/RtAudio.h#L49-L52, whereas `meson.build` says `6.0.0beta1` https://github.com/thestk/rtaudio/blob/0e11b50a7ba85875d09d60c097092d1d4bd760b2/meson.build#L2

**Comment by dvzrv** [comment]
For 6.0.0 at least this appears fixed with https://github.com/thestk/rtaudio/commit/2f17e135967e63f011a04f6e4a09b9b9a758a044

**Comment by garyscavone** [comment]
Yes, this was fixed, very soon after committing the version update, so I don't understand how this would persist.

**Comment by dvzrv** [comment]
Now fixed with 6.0.1. This can be closed.
